lib/loadLoader.js

```diff
@@ -27,12 +31,16 @@ module.exports = function loadLoader(loader, callback) {
 			return callback(e);
 		}
 		if(typeof module !== "function" && typeof module !== "object")
-			throw new Error("Module '" + loader.path + "' is not a loader (export function or es6 module))");
+			return callback(new LoaderRunnerError(
+        "Module '" + loader.path + "' is not a loader (export function or es6 module)"
```

Comment:
fix your spacing

Maybe add a `.editorconfig` file to help here.

Reply:
done, in latest push